code_challenges/code_wars/longest_collatz_sequence.py

```diff
@@ -21,7 +21,7 @@ def find_longest_length(numbers):
     winner = 0
     winner_length = 0
     for number in numbers:
-        if len(numbers_with_sequence[number]) > winner_length:
+        if numbers_with_sequence[number] > winner_length:
             winner = number
-            winner_length = len(numbers_with_sequence[number])
+            winner_length = numbers_with_sequence[number]
```

Comment:
Use the max function on `numbers_with_sequence` instead.  Note you'll have to use the `key` kwarg.

Reply:
can i use max since? if there are 2 numbers with the same length then it goes based off of which one is first in the list and i thought dictionaries dont always keep the same order

Reply:
> can i use max since? if there are 2 numbers with the same length then it goes based off of which one is first in the list

Your right, _since you are using a dictionary when you don't need one_, you can't be sure of the order. (even though in Python 3 you can...)